components/dockmanager/state.md

```diff
@@ -0,0 +1,222 @@
+---
+title: State
+page_title: DockManager - State
+description: Save, load, change the DockManager for Blazor state - docking, undocking, resizing and so on.
+slug: dockmanager-state
+tags: telerik,blazor,dockmanager,state,save,load,layout,set,change,management
+published: True
+position: 5
```

Comment:
The state is a more complex concept, so it should come after the article about panes and dock types. The customer should first know how to configure the component declaratively, and then how to use the state.